static/script.js

```diff
@@ -72,24 +74,37 @@ function enableAutoRefresh () { // eslint-disable-line no-unused-vars
 }
 
 function unfoldAll () { // eslint-disable-line no-unused-vars
-  window.setInterval(unfoldButton, 100)
   $('#unfold-all-li').removeClass('d-none')
-  let allUnfoldIds = $('#refs_list').attr('content').replace(/[\[\]'"\ ]/gm, '').split(',').sort().toString() // eslint-disable-line
+  let allIds = $('#refs_list').attr('content')                                  // eslint-disable-line
```

Comment:
Do you still need `// eslint-disable-line`? Splitting methods chain over multiple lines was meant to deal with linter complaining about too long line.

Reply:
yes I need it, without comments standard linter drops errors :(

Reply:
That's pity. Maybe we need to tweak linter a bit, to be less restrictive.